v2/component_analyses.yaml

```diff
@@ -0,0 +1,288 @@
+        pname:
+          type: array
+          items:
+            type: string
+        version:
+          type: array
+          items:
+            type: string
+        declared_licenses:
+          type: array
+          items:
+            type: string
+        licenses:
+          type: array
+          items:
+            type: string
+        last_updated:
+          type: array
+          items:
+            type: number
+            format: float
+        vulnerability_ids:
+          type: array
+          items:
+            type: string
+        description:
+          type: array
+          items:
+            type: string
+      description: Component version details
+    Package:
+      title: Component Package Specific Details
+      type: object
+      properties:
+        libio_dependents_repos:
+          type: array
+          items:
+            type: string
+        libio_latest_release:
+          type: array
+          items:
+            type: string
+        gh_prs_last_year_opened:
+          type: array
+          items:
+            type: integer
+        libio_latest_version:
+          type: array
+          items:
+            type: string
+        last_updated:
+          type: array
+          items:
+            type: number
+            format: float
+        gh_subscribers_count:
+          type: array
+          items:
+            type: number
+        gh_stargazers:
+          type: array
+          items:
+            type: number
+        libio_dependents_projects:
+          type: array
+          items:
+            type: string
+        gh_forks:
+          type: array
+          items:
+            type: integer
+        libio_usedby:
+          type: array
+          items:
+            type: string
+        gh_issues_last_month_closed:
+          type: array
+          items:
+            type: integer
+        ecosystem:
+          type: array
+          items:
+            type: string
+        gh_prs_last_year_closed:
+          type: array
+          items:
+            type: integer
+        gh_issues_last_year_opened:
+          type: array
+          items:
+            type: integer
+        gh_prs_last_month_closed:
+          type: array
+          items:
+            type: integer
+        gh_prs_last_month_opened:
+          type: array
+          items:
+            type: integer
+        vertex_label:
+          type: array
+          items:
+            type: string
+        gh_contributors_count:
+          type: array
+          items:
+            type: integer
+        gh_issues_last_month_opened:
+          type: array
+          items:
+            type: integer
+        gh_issues_last_year_closed:
+          type: array
+          items:
+            type: integer
+        latest_version:
+          type: array
+          items:
+            type: integer
+        gh_open_issues_count:
+          type: array
+          items:
+            type: integer
```

Comment:
I still see lots of unused/unnecessary fields in CA response, can you strip those things as well?

Reply:
Ack!